exporter/trace/go.mod

```diff
@@ -31,8 +33,11 @@ require (
 	github.com/googleapis/gax-go/v2 v2.2.0 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	go.opencensus.io v0.23.0 // indirect
+	go.opentelemetry.io/collector/model v0.49.0 // indirect
```

Comment:
This is somewhat unfortunate... The collector and the go SDK have two different packages for semconv.  E.g. https://github.com/open-telemetry/opentelemetry-go/tree/main/semconv/v1.9.0 vs https://github.com/open-telemetry/opentelemetry-collector/tree/main/semconv/v1.9.0.  I think we should probably use `go.opentelemetry.io/otel/semconv/v1.9.0`, rather than the collector one.  It seems less strange for the collector to depend on the SDK than for the SDK exporter to depend on the collector.

Reply:
Ah, good point.  I didn't even realize I was pulling the wrong one....

Reply:
Hmm, I'm having trouble finding a published version from the SDK -

https://pkg.go.dev/go.opentelemetry.io/otel/semconv/v1.9.0?tab=versions

```shell
$ go get go.opentelemetry.io/otel/semconv/v1.9.0
go get: module go.opentelemetry.io/otel@upgrade found (v1.6.3), but does not contain package go.opentelemetry.io/otel/semconv/v1.9.0
```

Should I move back to a version that exists?   How often does go publish these?

Reply:
Migrated to v1.7.0 (latest released version) of semconv.